### PR TITLE
Use yes/no option labels in tally reports

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview

Previously we just used "Yes" and "No" labels instead of the provided labels, which could be inconsistent with what's printed on the ballot.

## Demo Video or Screenshot
See snapshots in diff
## Testing Plan

Updated automated tests, manually tested in VxAdmin

Note: integration test currently needs a snapshot update, but there's a [bug in Playwright](https://github.com/microsoft/playwright/issues/32186) so I can't download Chromium at this moment

## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: apps/scan/backend/src/__image_snapshots__/polls-closed-report-1.png
Comment (jonahkagan) [inline apps/scan/backend/src/__image_snapshots__/polls-closed-report-1.png:1]: Imperceptible diff on this one - doesn't seem related to the logical change

**Comment by carolinemodic** [comment]
@jonahkagan looks like the snapshots are still messed up

**Comment by jonahkagan** [comment]
@carolinemodic ok finally got the snapshots fixed

**Review by carolinemodic — APPROVED** [review]
